### PR TITLE
Use version in package status

### PR DESCRIPTION
**Pull request by TerryHowe**
Set Status.Version to human readable version

```
NAME      PACKAGE   AGE     STATE       VERSION   DETAIL
my-test   Test      5m47s   installed   v0.1.0 
```

### CODE REVIEW

File: controllers/packagebundlecontroller_controller.go [hunk 1]
```diff
@@ -123,7 +123,8 @@ func (r *PackageBundleControllerReconciler) Reconcile(ctx context.Context, req c
 
 	latestBundle, err := r.bundleManager.LatestBundle(ctx, abc.Spec.Source.BaseRef())
 	if err != nil {
-		return result, err
+		r.Log.Error(err, "Error getting latest bundle")
```
Comment (ewollesen) [inline controllers/packagebundlecontroller_controller.go:126]: Why this change? Was the error not being logged previously?

Comment (ewollesen) [reply controllers/packagebundlecontroller_controller.go:126]: I can merge regardless, but I'm curious.

File: controllers/package_controller.go [hunk 2]
```diff
@@ -128,7 +128,7 @@ func (r *PackageReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 			return ctrl.Result{RequeueAfter: retryLong}, err
 		}
 
-		managerContext.Source, err = bundle.FindSource(managerContext.Package.Spec.PackageName, managerContext.Package.Spec.PackageVersion)
+		managerContext.Source, managerContext.Version, err = bundle.FindSource(managerContext.Package.Spec.PackageName, managerContext.Package.Spec.PackageVersion)
```
Comment (lewisdiamond) [inline controllers/package_controller.go:131]: I think you can avoid passing down the `version.Name` from `FindSource` all the way to here by just using `managerContext.Package.Spec.PackageVersion` if a source was found.

Comment (lewisdiamond) [reply controllers/package_controller.go:131]: It's my opinion that if the version installed was specified as a `sha256:tag`, we should display that. In the current implementation, it would use the `version.Name`, but if you use `managerContext.Package.Spec.PackageVersion`, it would display the `sha256:tag` if that's what was specified (those cases are going to be rare of course).

File: controllers/packagebundlecontroller_controller.go (hunk 1 above)
Comment (TerryHowe) [reply controllers/packagebundlecontroller_controller.go:126]: I didn't mean to include that change.

File: controllers/package_controller.go (hunk 2 above)
Comment (TerryHowe) [reply controllers/package_controller.go:131]: The Package.Spec.PackageVersion is the requested version, not necessarily the installed version. The sha is in managerContext.Source

Comment (TerryHowe) [reply controllers/package_controller.go:131]: I added version here from the bundle `name` because it is the human readable version. Otherwise the user would just have a sha and no idea what that releates to without looking it up in the bundle

**Review by ewollesen — APPROVED** [review]
